docs/docs/how-to/plugins-and-themes/using-a-plugin-in-your-site.md

```diff
@@ -1,56 +1,93 @@
 ---
-title: Using a Plugin in Your Site
+title: Add a Plugin to Your Site
```

Comment:
Should this doc change be part of a separate PR?

Reply:
I think it's okay to leave in here, since I updated it as part of the "how to write a how-to guide" contributing doc. To make something that fits the template.

Reply:
Okay 👍